packages/aws-cdk/README.md

```diff
@@ -746,6 +746,32 @@ NOTICES
 If you don’t want to see a notice anymore, use "cdk acknowledge <id>". For example, "cdk acknowledge 16603".
 ```
 
+### `cdk migrate`
+
+⚠️**CAUTION**⚠️ CDK migrate is currently experimental. We make no guarantees about the outcome or stability of the Command or it's output.
+
+Generates a CDK application from an existing Cloudformation template in any of the CDK supported languages. 
+Required Arguments:
+* `--from-path <my_file_path>` - Takes the relative filse path to the JSON or YAML template of your choice
+* `--stack-name <my_stack_name>` - Used to name both the CDK application and stack. 
+Optional Arguments:
+* `--output-path <my_output_path>` - file path to where the file should be generated. 
+  Default is the current working directory
+* `--language <language>` - Which CDK supported language should be generated [typescript, python, csharp, java, go]
+  default is typesript
+```console
+$ # generate a typescript application from template.json in the local directory
+$ cdk migrate --from-path ./template.json --stack-name MyAwesome
+
+$ # generate a python application from template.json in the local directory
+$ cdk bootstrap migrate --from-path ./template.json --stack-name MyAwesome
+```
+
+In order for the generated stack to be immediatly deployable, that stack must either already exist in that region/account 
```

Comment:
put a header above this that says "Limitations" like other commands have done (import)

Reply:
Overall this section is pretty confusing, but I am struggling to come up with better phrasing here. I imagine customers will read this and ask, what does it mean to be immediately deployable? How do I know if there is any overlap between existing resources and resources defined in the input template?

An idea to improve clarity would be to separate this into describing the two use cases that work: 1. deploying to an environment with a deployed stack that matches the input template. 2. deploy to an environment with no deployed resources that overlap with your template definition. 
Then finally, include the limitations section saying if you try to deploy a stack generated by CDK migrate to an environment that meets any of the following criteria, deployment will fail.
- resources with the same physical ids already exist
- a partially-matching stack already exists.